scripts/check-for-update.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
```

Comment:
Add `set -e` so the script fails if `get-latest-version.sh` fails.

Reply:
Updated!